lib/skate/detours/snapshot_serde.ex

```diff
@@ -82,17 +87,16 @@ defmodule Skate.Detours.SnapshotSerde do
 
   defp validate_serialized_snapshot(%Detour{id: id} = detour) do
     serialized_snapshot = serialize_snapshot(detour)
-    {matches, diff, scoped_state} = compare_snapshots(detour, serialized_snapshot)
+    {matches, diff} = compare_snapshots(detour, serialized_snapshot)
 
     if matches do
```

Comment:
This if/else can be polished into just an `if` that handles logging now that both cases return `serialized_snapshot`